src/main/resources/static/styles.css

```diff
@@ -20,6 +15,8 @@
 
 .nav-link {
     font-size: 2rem;
+    padding-top:1rem;
```

Comment:
these helped the spacing on navbar after making changes for the button functionality